conans/client/command.py

```diff
@@ -888,7 +888,7 @@ def build(self, *args):
                                      update=args.update, generators=args.generator,
                                      no_imports=args.no_imports,
                                      lockfile=args.lockfile,
-                                     lockfile_out=args.lockfile_out)
+                                     lockfile_out=args.lockfile_out, conf=args.conf_host)
```

Comment:
How is this related?

Reply:
This was to fix `test_apple_framework_xcode` test that passes the generator through the command line with `build . -c tools.cmake.cmaketoolchain:generator=Xcode` but the argument was not taken in the build command.